pkgs/top-level/all-packages.nix

```diff
@@ -9551,6 +9551,8 @@ with pkgs;
 
   jet = callPackage ../development/tools/jet { };
 
+  jetporch = callPackage ../tools/admin/jetporch { };
+
```

Comment:
I think the new best practice is to have packages in pkgs/by-name.
For jetporch, the location would be `pkgs/by-name/je/jetporch/package.nix`.
That way you do not need to add the package to all-packages.nix

Reply:
So would I put the package in `pkgs/by-name/je/jetporch/package.nix` instead of `pkgs/tools/admin/jetporch/default.nix`?

Reply:
Done! 7ab4cae0c31fd3f02a42ca31aedf8996f31b8738